### ISSUE TITLE
No Tests are written at all

### ISSUE THREAD

**Issue by AceMgy** [evewspace/eve-wspace#228]
So I was messing around trying to make my first commit and I went to run the tests in django to make sure I didn't break anything.  I noticed that absolutely nobody has touched any */tests.py file.

I've only run through some django tutorials and made a simple database of my own, but I found making test functions really useful for my workflow.  Is it worth worrying about writing these or does nobody really need them?

**Comment by Zumochi** [comment]
It would indeed be useful if they're there. If you feel like writing them :)

**Comment by AceMgy** [comment]
Yeah I debated even complaining about it because you know what happens to those who complain...

I'll concentrate on other things and make sure I write tests for any new functions I create.  Might come back to this if I'm really bored.
